ui/app/utils/open-exec-url.js

```diff
@@ -0,0 +1,3 @@
+export default function openExecUrl(url) {
+  window.open(url, '_blank', 'width=973,height=490,location=1');
```

Comment:
I wonder if down the road we can do something fancier here by detecting native resolution or something.